app/controllers/activity_executions_controller.rb

```diff
@@ -5,8 +5,7 @@ class ActivityExecutionsController < ApplicationController
   load_and_authorize_resource through: :activity
 
   def index
-    @spots = Spot.all.order(:name)
-
+    @spots = Spot.all.includes(:fields).order(Arel.sql("LOWER(spots.name->>'de')"))
```

Comment:
avoid N+1 query, order by JSON. Limitation: the ordering works not with fallbacks. since most will have de i will go with that